TrackingTools/GsfTracking/python/BwdAnalyticalPropagator_cfi.py

```diff
@@ -1,11 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
-import copy
+#import copy
```

Comment:
This can be removed

Reply:
> @jeongeun you forgot this line

Thank you Silvio for checking: I missed it
I remove the "+1" while waiting for this (simple) adjustment